src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Why this field is low letter case? Please re-read google, link: https://mate-academy.github.io/style-guides/java/java.html